src/java.base/unix/native/libnet/PlainDatagramSocketImpl.c

```diff
@@ -2096,11 +2107,18 @@ static void mcast_join_leave(JNIEnv *env, jobject this,
 #endif
 
         /* Join the multicast group */
-        if (setsockopt(fd, IPPROTO_IPV6, (join ? ADD_MEMBERSHIP : DRP_MEMBERSHIP),
-                       (char *) &mname6, sizeof (mname6)) < 0) {
+        res = setsockopt(fd, IPPROTO_IPV6, (join ? ADD_MEMBERSHIP : DRP_MEMBERSHIP),
+                       (char *) &mname6, sizeof (mname6));
 
```

Comment:
Seems to be an extraneous space after sizeof here.

Reply:
A general question. Is "__APPLE__" the preferred macro name or MACOSX? Not a big deal but MACOSX looks slightly more common. [Seems github has removed the underscores from APPLE]

Reply:
thanks ... I'll attend to the spacing 
I used __APPLE__ as it was already in place for a change to set socket buffer size